A/ADIOS2/build_tarballs.jl

```diff
@@ -7,24 +7,40 @@ version = v"2.7.1"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/ornladios/ADIOS2/archive/refs/tags/v2.7.1.tar.gz", "c8e237fd51f49d8a62a0660db12b72ea5067512aa7970f3fcf80b70e3f87ca3e")
+    ArchiveSource("https://github.com/ornladios/ADIOS2/archive/refs/tags/v2.7.1.tar.gz", "c8e237fd51f49d8a62a0660db12b72ea5067512aa7970f3fcf80b70e3f87ca3e"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
 cd ADIOS2-2.7.1
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/shlwapi.patch
 mkdir build
 cd build
-# if [[ "$target" == x86_64-w64-mingw32 ]]; then
-#     mpiopts="-DMPI_HOME=$prefix -DMPI_GUESS_LIBRARY_NAME=MSMPI -DMPI_C_LIBRARIES=msmpi64 -DMPI_CXX_LIBRARIES=msmpi64"
-# elif [[ "$target" == *-mingw* ]]; then
-#     mpiopts="-DMPI_HOME=$prefix -DMPI_GUESS_LIBRARY_NAME=MSMPI"
-# else
-#     mpiopts=
-# fi
+if [[ "$target" == *-apple-* ]]; then
+    # Set up a wrapper script for the assembler. GCC's assembler
+    # output isn't accepted by the LLVM assembler.
+    as=$(which as)
+    mv "$as" "$as.old"
+    export AS="$as.old"
+    ln -s "$WORKSPACE/srcdir/scripts/as.llvm" "$as"
+fi
 mpiopts=
-cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake -DCMAKE_BUILD_TYPE=Release -DADIOS2_USE_Fortran=OFF -DADIOS2_BUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF ${mpiopts} ..
+winopts=
+if [[ "$target" == x86_64-w64-mingw32 ]]; then
+    # cmake's auto-detection doesn't work on Windows.
+    # The SST and Table ADIOS2 components don't build on Windows
+    # (reported in <https://github.com/ornladios/ADIOS2/issues/2705>)
+    mpiopts="-DMPI_HOME=$prefix -DMPI_GUESS_LIBRARY_NAME=MSMPI -DMPI_C_LIBRARIES=msmpi64 -DMPI_CXX_LIBRARIES=msmpi64"
+    winopts="-DADIOS2_USE_SST=OFF -DADIOS2_USE_Table=OFF"
+elif [[ "$target" == *-mingw* ]]; then
+    mpiopts="-DMPI_HOME=$prefix -DMPI_GUESS_LIBRARY_NAME=MSMPI"
+    winopts="-DADIOS2_USE_SST=OFF -DADIOS2_USE_Table=OFF"
+fi
+# We disable Fortran because this would require building many more
+# versions of the library
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake -DCMAKE_BUILD_TYPE=Release -DADIOS2_USE_Fortran=OFF -DADIOS2_BUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF ${mpiopts} ${winopts} ..
 make -j${nproc}
 make -j${nproc} install
 """
```

Comment:
Now license files aren't installed automatically:
```suggestion
install_license Copyright.txt LICENSE
"""
```

Reply:
Done.